extensions/ql-vscode/src/interface-utils.ts

```diff
@@ -116,11 +117,38 @@ export function tryResolveLocation(
  * Uses a content security policy that only loads the given script.
  */
 export function getHtmlForWebview(
+  ctx: ExtensionContext,
   webview: Webview,
-  scriptUriOnDisk: Uri,
-  stylesheetUrisOnDisk: Uri[],
-  allowInlineStyles: boolean
+  view: 'results' | 'compare' | 'remote-queries',
+  {
+    allowInlineStyles,
+    includeCodicons
+  }: {
+    allowInlineStyles?: boolean;
+    includeCodicons?: boolean;
+  } = {
+      allowInlineStyles: false,
+      includeCodicons: false
+    }
 ): string {
+  const scriptUriOnDisk = Uri.file(
+    ctx.asAbsolutePath('out/webview.js')
+  );
+
+  const stylesheetUrisOnDisk = [
+    Uri.file(ctx.asAbsolutePath('out/webview.css'))
+  ];
+
+  if (includeCodicons) {
```

Comment:
As you have noticed in #1478, I think we should always include the codicons and will do it in that PR. However, to include the codicons the path should also be included in the `localResourceRoots`, so that would require duplicating [this line](https://github.com/github/vscode-codeql/blob/45da1e0f1fa845c6f7b48d5a5ca52b69a4680646/extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts#L120) to the other two interface managers. I didn't really think that would be worth it, especially since that part is going to be replaced anyway in #1478.